src/backend/impl/src/mappings/proposal.rs

```diff
@@ -48,11 +36,9 @@ impl From<backend_api::ReviewPeriodState> for ReviewPeriodState {
 impl From<Proposal> for backend_api::Proposal {
     fn from(value: Proposal) -> Self {
         backend_api::Proposal {
-            title: value.title,
-            nervous_system: value.nervous_system.into(),
+            nervous_system: value.nervous_system.clone().into(),
             state: value.state.into(),
-            proposed_at: value.proposed_at.to_string(),
-            proposed_by: value.proposed_by,
+            proposed_at: value.proposed_at().unwrap().to_string(),
```

Comment:
What do you think about implementing `TryFrom` here and then handling the error in the controller so we get a proper response instead of a panic?

Reply:
Makes sense, done